llvm/include/llvm/IR/StructWideningUtils.h

```diff
@@ -0,0 +1,30 @@
+//===--- StructWideningUtils.h - Utils for widening/narrowing struct types ===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_IR_STRUCTWIDENINGUTILS_H
+#define LLVM_IR_STRUCTWIDENINGUTILS_H
+
+#include "llvm/IR/DerivedTypes.h"
+
+namespace llvm {
+
+/// A helper for converting structs of scalar types to structs of vector types.
+/// Note: Only unpacked literal struct types are supported.
+Type *ToWideStructTy(StructType *StructTy, ElementCount EC);
```

Comment:
```suggestion
Type *toWideStructTy(StructType *StructTy, ElementCount EC);
```

keep cases between new function consistent